spec/client/client/SetupLinksAndPushStateSpec.js

```diff
@@ -79,47 +91,70 @@ describe("SetupLinksAndPushState", function() {
         });
 
         it("sets up application links to set ClientRequest as from link click", function() {
-            $fixture.find("a.applink").click();
```

Comment:
i'm going to investigate why jquery click doesn't bubble to native click handler